Sources/MuxUploadSDK/Public API/UploadManager.swift

```diff
@@ -89,18 +77,51 @@ public final class UploadManager {
         }
     }
     
+    /// Adds an ``UploadsUpdatedDelegate`` You can add as many of these as you like
+    public func addUploadsUpdatedDelegate(_ delegate: (any UploadsUpdatedDelegate)) {
```

Comment:
```suggestion
    public func addUploadsUpdatedDelegate<Delegate: UploadsUpdatedDelegate>(_ delegate: Delegate) {
```

Reply:
Dropping `any` here will decrease runtime overhead without any syntactic changes to the call site (i.e. this looks the same either way to clients)

Reply:
More on this here if curious: https://www.hackingwithswift.com/swift/5.6/existential-any